spec/rubocop/cop/rspec/verified_double_reference_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::VerifiedDoubleReference do
+  context 'when EnforcedStyle is const' do
+    let(:cop_config) do
+      { 'EnforcedStyle' => 'const' }
+    end
+
+    it 'does not flag a violation when using a const reference' do
+      expect_no_offenses('instance_double(ClassName)')
+    end
+
+    it 'flags a violation when using a string reference' do
+      expect_offense(<<-RUBY)
+        instance_double("ClassName")
+                        ^^^^^^^^^^^ Prefer `const` references for verfied doubles.
```

Comment:
I think users would know better how to resolve the error is we didn’t use `const` in the instructions.
```suggestion
                        ^^^^^^^^^^^ Prefer `constant` references for verfied doubles.
```